components/color-picker/index.jsx

```diff
@@ -419,7 +419,16 @@ class ColorPicker extends React.Component {
 					iconPosition="right"
 					iconVariant="more"
 					id={`slds-color-picker__summary-button-${this.generatedId}`}
-					label={<Swatch color={this.state.currentColor} labels={labels} />}
+					label={
+						<div>
+							<span className="slds-assistive-text">
```

Comment:
This mirrors the way assistive text is being added by Button component.  But we need to add it manually here since the label is a node instead of a string.

Reply:
@kevinparkerson thanks for approving.  Much appreciated!  Would you also know how to remove the cla:missing tag?  I signed the confirmation form already but it's not being picked up.

Reply:
Great find on this Button label @kmmkwong